modules/040-control-plane-manager/openapi/config-values.yaml

```diff
@@ -171,7 +171,13 @@ properties:
             # Avoid trailing slash
             pattern: ^[a-zA-Z0-9_/.-]+[a-zA-Z0-9_.-]$
             default: /var/log/kube-audit
-
+      encryptingAtRestEnabled:
```

Comment:
What about renaming this parameter to `encryptionEnabled`?

Reply:
yep, it is good naming